xbmc/pvr/PVRManager.cpp

```diff
@@ -133,6 +140,19 @@ void CPVRManager::Announce(AnnouncementFlag flag, const char *sender, const char
     TriggerEpgsCreate();
     TriggerTimersUpdate();
   }
+  else if (strcmp(message, "OnSleep") == 0)
+  {
+    for (auto updateJob : m_pendingUpdates)
```

Comment:
Apparently `auto foo = somepointer;` is correct, see https://stackoverflow.com/questions/12773257/does-auto-type-assignments-of-a-pointer-in-c11-require. When using references you have to specify it implicitly though, e.g. `auto &foo = somevector;`.